CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.6.2] - 2021-03-10
+### Fixed
+* [#545](https://github.com/plotly/dash-bio/pull/545) `sideByside` prop fix for NglMoleculeViewer when `ALL` chains are visualized.
+### Added
+* [#546](https://github.com/plotly/dash-bio/pull/546) Added shapes and isosurfaces props to 3dMoleculeViewer to enable rendering additional features on the molecule.
```

Comment:
These numbers aren't correct - both of these are in this PR, right?

BTW just a followup on our slack discussion: typically when we're adding features we'll just add an `## Unreleased` heading, with no date or release number, and then replace it with the actual release number and date during the actual release process.

Reply:
Thanks for pointing it out, those were WIP numbers and I've fixed them in 6e5c669, along with the version.